test/FunctionalTests/Server/ClientStreamingMethodTests.cs

```diff
@@ -275,8 +275,6 @@ async Task<CounterReply> AccumulateCount(IAsyncStreamReader<CounterRequest> requ
             var responseTask = Fixture.Client.SendAsync(httpRequest, HttpCompletionOption.ResponseHeadersRead);
 
             // Assert
-            Assert.IsFalse(responseTask.IsCompleted, "Server should wait for client to finish streaming");
```

Comment:
Why is this removed?

Reply:
It's a bad assert. The client stream is being sent immediately.